vms/platformvm/api/static_service.go

```diff
@@ -88,10 +88,6 @@ type Staker struct {
 	EndTime   json.Uint64 `json:"endTime"`
 	Weight    json.Uint64 `json:"weight"`
 	NodeID    ids.NodeID  `json:"nodeID"`
-
-	// Deprecated: Use Weight instead
-	// TODO: remove [StakeAmount] after enough time for dependencies to update
-	StakeAmount *json.Uint64 `json:"stakeAmount,omitempty"`
```

Comment:
Should this part of the release notes, just in case ™️  Deprecating and removing is not the same really.